### ISSUE TITLE
Same As with Interval Demotion and Interval Promotion Disabled

### ISSUE THREAD

**Issue by JSRankins** [cqframework/clinical_quality_language#341]
@brynrhodes 

In CQL-to-ELM v1.2.20, the following CQL was translated without error:

```
["Encounter, Performed"] EP
     where EP.relevantPeriod same as EP.authorDatetime
```

CQL-to-ELM translator v13 now provides an error when this same CQL is provided and Interval Demotion and Interval Promotion are disabled.

Per previous discussions, implicit conversion would occur for operators that would have previously required Interval Promotion or Interval Demotion to be enabled for backwards compatibility. Should this CQL be mapped over to something equivalent without enabling Interval Promotion or Interval Demotion.

**Comment by brynrhodes** [comment]
@JSRankins yes, that behavior is a little different now. The analysis and discussion focused on only allowing operations where the semantics are clear, and the notion of comparing an interval to a date time for equality is unclear (is the author comparing the datetime to the start of the interval or the end?). As such, the resolution for comment STU3:167 explicitly excluded `same as` from the list of operations that would support mixed invocation.


So this is a non-backwards compatible change, but a worthwhile one to remove the ambiguity, and based on regression, is not a pattern that was actually used in any production expressions.


Does that help?

**Comment by JSRankins** [comment]
@brynrhodes We are good with this response if the measure development community is. Let's discuss it at the 10 AM meeting coming up.
